swap/src/bin/swap.rs

```diff
@@ -327,6 +327,40 @@ async fn init_monero_wallet(
     Ok((monero_wallet, monero_wallet_rpc_process))
 }
 
+async fn qr_code(deposit_address: &bitcoin::Address) -> Result<String> {
+    let code = QrCode::new(deposit_address.to_string())?;
+    let qr_code = code
+        .render::<unicode::Dense1x2>()
+        .dark_color(unicode::Dense1x2::Light)
+        .light_color(unicode::Dense1x2::Dark)
+        .build();
+    let first_line = "QR Code:".to_string() + qr_code.lines().next().unwrap() + "\n";
```

Comment:
Do we have to say "QR Code:"? It is kind of obvious IMO and it would make this implementation a lot simpler because we could just do: `qr_code.lines().join("\n")`

Reply:
Agree. Removed it. We don't need even to join it manually the build method of qrcode crate already returns it as a String.